lib/AcqUnitFilter/AcqUnitFilter.js

```diff
@@ -15,8 +18,12 @@ const AcqUnitFilter = ({ acqUnits, ...rest }) => {
   );
 };
 
+AcqUnitFilter.manifest = Object.freeze({
+  acqUnits: acqUnitsResource,
+});
+
 AcqUnitFilter.propTypes = {
-  acqUnits: acqUnitsShape,
+  resources: PropTypes.object.isRequired,
 };
 
-export default AcqUnitFilter;
+export default stripesConnect(AcqUnitFilter);
```

Comment:
this is better to do in a separate container component

Reply:
added container